cibyl/orchestrator.py

```diff
@@ -192,6 +158,53 @@ def setup_sources(self):
                         if source.enabled:
                             source.setup()
 
+    def load_features(self):
+        """Read user-requested features and setup the right argument to query
+        the information for them."""
+        features_loader = FeaturesLoader()
```

Comment:
Have this instantiation after the 'user_features' check, you will avoid all those problems than can occur from reflection that way.

Reply:
Good catch, thanks!